std/assembly/date.ts

```diff
@@ -1,7 +1,22 @@
-import { E_VALUEOUTOFRANGE } from "util/error";
+import { E_INVALIDDATE } from "util/error";
 import { now as Date_now } from "./bindings/Date";
 
+// @ts-ignore: decorator
+@inline const
```

Comment:
Should we make these @lazy as well? Or is that implied by @inline?

Reply:
No, const globals usually not needed lazy due to it will always removed by binaryen